digits/templates/socketio.html

```diff
@@ -3,7 +3,7 @@
 <!-- Scripts for communicating with the SocketIO server -->
 <script type="text/javascript" src="/static/js/3rdparty/socket.io-1.4.5.min.js"></script>
 <script>
-var socket = io.connect('http://' + document.domain + ':' + location.port + "{{namespace}}");
+var socket = io.connect(location.protocol + '//' + document.domain + ':' + location.port + "{{namespace}}");
```

Comment:
Why don't we go whole hog and use `location.origin`? Browser support looks fine.
http://www.w3schools.com/jsref/prop_loc_origin.asp